spec/helper.rb

```diff
@@ -218,26 +222,17 @@ def current_database
   end
 end
 
-RSpec::Matchers.define :match_sql do |expected_sql|
-  match do |actual_sql|
-    normalize_sql(actual_sql) == normalize_sql(expected_sql)
-  end
-
-  failure_message do |actual_sql|
-    <<~MSG
-      Expected SQL to match:
-        #{normalize_sql(expected_sql)}
-
-      But got:
-        #{normalize_sql(actual_sql)}
-    MSG
+QueryUnderTest = Struct.new(:sql, :connection) do
```

Comment:
In a subsequent PR, I intend to add an `#explain` interface to this, which is why it's an actual struct class and not just a query string.